### ISSUE TITLE
send user utcOffset when uploading csv data

### ISSUE THREAD

**Issue by rfldong** [dongrote/ketonechart#15]
Use `moment().utcOffset()` to send the users `utcOffset` along with their `.csv` file. Hopefully their KetoMojo blood test unit is configured for the same timezone as their browser.

**Comment by rfldong** [comment]
```
<script crossorigin src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.22.2/moment.min.js"></script>
```

**Comment by rfldong** [comment]
Fixed by b2735c4666c3310c4015155ac8b5d83b279912e6
